app/models/changeset.rb

```diff
@@ -84,13 +84,25 @@ def find_comparison
 
   def find_pull_requests
     numbers = commits.map(&:pull_request_number).compact
-    numbers.map { |num| PullRequest.find(repo, num) }.compact
+    numbers.map { |num| PullRequest.find(repo, num) }
+      .compact
+      .concat(find_pull_request_for_branch)
   end
 
   def cache_key
     [self.class, repo, previous_commit, commit].join('-')
   end
 
+  def find_pull_request_for_branch
+    # Bail if commit looks like a commit SHA
+    return [] if commit =~ /[0-9a-f]{40}/
+    org = repo.split("/", 2).first
+    GITHUB.pull_requests(repo, head: "#{org}:#{commit}")
```

Comment:
does not work without the `org:` ?

Reply:
https://github.com/octokit/octokit.rb/issues/865#issuecomment-280454776